crates/pathfinder/src/bin/pathfinder/config.rs

```diff
@@ -56,6 +56,16 @@ Examples:
     )]
     rpc_address: SocketAddr,
 
+    #[arg(long = "ws", long_help = "Enable WebSocket transport")]
+    ws: bool,
+
+    #[arg(
+        long = "ws.capacity",
+        long_help = "Capacity for WebSocket Senders",
```

Comment:
Users won't really know what Senders are in this context, so maybe something more like:
```suggestion
        long_help = "Maximum number of subscriptions per subscription type",
```

Reply:
I guess we're missing the env variable option here too:
`env = "PATHFINDER_RPC_WEBSOCKET_CAPACITY"`
:arrow_up: Sounds strange but the alternative I came up with is just so long:
`PATHFINDER_RPC_WEBSOCKET_SUBSCRIPTION_CAPACITY_PER_TYPE`

Reply:
How about adding `SENDER`?
As in maybe `PATHFINDER_RPC_WEBSOCKET_SENDER_CAPACITY`?

Reply:
Scratch that, it's too long...
`PATHFINDER_RPC_WEBSOCKET_CAPACITY` is it!